tests/Feature/ReceiveDocumentsWorkflowTest.php

```
@@ -0,0 +1,258 @@
+        // Act
+        $errorCaught = false;
+        try {
+            $proxyClient->getAllNewDocuments(50);
+        } catch (ApiException $e) {
+            $errorCaught = true;
+            $this->assertStringContainsString('Error receiving documents', $e->getMessage());
+        }
+
+        // Assert
+        $this->assertTrue($errorCaught);
+    }
```

PHPMD (2.15.0)

The method it_handles_error_during_document_retrieval is not named in camelCase. (undefined)

(CamelCaseMethodName)